android/app/src/main/java/io/locha/p2p/runtime/ChatServiceModule.java

```diff
@@ -150,11 +154,14 @@ public void getPeerId(Promise promise) {
      *
      * @see <a href="https://multiformats.io/multiaddr/">Multiaddr</a>
      */
-    @ReactMethod public void dial(String multiaddr) {
+    @ReactMethod public void dial(String multiaddr, Promise promise) {
+        Log.i(TAG, "dial: hereeeee");
         try {
             nativeDial(multiaddr);
+            promise.resolve(null);
         } catch (Exception e){
-            Log.e(TAG, "Couldn't dial address", e);
+            Log.e(TAG, "Couldn't dial address"+ e.toString());
```

Comment:
```suggestion
            Log.e(TAG, "Couldn't dial address", e);
```

This way logcat will show the stack trace too. `nativeDial` shouldn't fail, only when ChatService hasn't been started, any other error is reported though events.

Reply:
other string can be:
```java
 Log.e(TAG, "Couldn't dial la verga huevona", e)
```
:smile: